integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -45,7 +45,14 @@ <h1 class="heading">
                 {% endif %}
             </h1>
             {% if not page_form.instance.id or not page_form.instance.archived %}
-                <div class="flex flex-wrap grow justify-end gap-2">
+                <div class="flex flex-wrap grow justify-end gap-2 items-center">
+                    <span id="auto-save"
+                          class="{% if page_translation_form.instance.status != AUTO_SAVE %} hidden {% endif %}">
+                        <i icon-name="check"></i>
+                        {% blocktranslate trimmed with auto_save_date=page_translation_form.instance.last_updated %}
+                            Automatically saved on <span id="auto-save-time">{{ auto_save_date }}</span>
+                        {% endblocktranslate %}
+                    </span>
```

Comment:
> I would also move the span out of the block translation, but I don't have a very strong opinion about that.

@timobrembeck I took it from your previous comment that it's bad practice to split up translation segments. Does this apply here too or is it a different situation because the span is at the end of the translation segment? :thinking: What would you consider to be the better practice?

Reply:
I think that keeping the sentence intact is definitely more important than extracting the `<span>`, but I'm not sure if @david-venhoff really suggested to split it up. There are also other possibilities to assemble the span outside of the translation block, e.g. maybe this could work? (but didn't test it)
```
{% blocktranslate trimmed with auto_save_date="<span id=\"auto-save-time\">"|add:page_translation_form.instance.last_updated|add:"</span>"|safe %}
    Automatically saved on {{ auto_save_date }}
{% endblocktranslate %}
```

Reply:
Except maybe if we turn it into two different sentence segments by e.g. separating them with a colon:
```
{% trans "Automatically saved on:" %} <span id="auto-save-time">{{ page_translation_form.instance.last_updated }}</span>
```

Reply:
> Except maybe if we turn it into two different sentence segments by e.g. separating them with a colon:

Yeah, I like that!

The first solution also worked, but djlint complained, that allegedly the span was an orphan, which I don't think it actually is. But if we want to go with the second idea anyway, it's not relevant anymore.